server/api/utils/query-builder.spec.js

```diff
@@ -0,0 +1,135 @@
+const { stub } = require('sinon');
+
+const Abstract = require('../../models/abstract');
+const expressFixtures = require('../../spec/fixtures/express');
+const { expect } = require('../../spec/utils');
+const { filter: filterFactory, queryBuilder: queryBuilderFactory } = require('./query-builder');
+
+describe('Query builder utilities', function() {
```

Comment:
No test that the filter function correctly update the query ?
Maybe because that would have been too complicated ? Or because it will ultimately be tested through API tests ?